environment.yml

```diff
@@ -22,5 +22,8 @@ dependencies:
     - richdem
     - scikit-gstat
     - pytransform3d
+    - https://github.com/GlacioHack/GeoUtils/tarball/master
```

Comment:
We should check that if we have a version of GeoUtils installed already in editable mode, it does not get overwritten with this environment.

Reply:
This works now with the latest commits.